mycore-base/src/main/java/org/mycore/datamodel/metadata/MCRMetadataManager.java

```diff
@@ -212,9 +211,7 @@ public static void create(final MCRDerivate mcrDerivate) throws MCRPersistenceEx
         fireEvent(mcrDerivate, null, MCREvent.CREATE_EVENT);
 
         // add the link to metadata
-        final MCRMetaDerivateLinkID der = new MCRMetaDerivateLinkID();
-        der.setReference(mcrDerivate.getId().toString(), null, mcrDerivate.getLabel());
-        der.setSubTag("derobject");
+        final MCRMetaDerivateLinkID der = MCRMetaDerivateLinkIDFactory.getInstance().getDerivateLink(mcrDerivate);
```

Comment:
this has changed from link to derobject, we need:
-> better documentation
-> better naming